<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iHouse - dashboard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Pokój 1</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="ustaw_temp1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>15.000000000000000</double>
      </property>
      <property name="maximum">
       <double>30.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QTextBrowser" name="kontrola_dymu1">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>260</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="wl_swiatlo1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Światło</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="limit_dymu1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="wl_wentylacja1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Wentylacja</string>
      </property>
     </widget>
     <widget class="QLabel" name="ustaw_temperature1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ustaw temperature:</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>-10</y>
        <width>20</width>
        <height>481</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="stezenie_dymu1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dopuszczalne stężenie dymu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="aktualna_temperatura1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aktualna temperatura:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="master_temp_wysw1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value" stdset="0">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QTextBrowser" name="czujnik_ruchu1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="czujnik_okna1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="czujnik_drzwi1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Pokój 2</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="ustaw_temp2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>15.000000000000000</double>
      </property>
      <property name="maximum">
       <double>30.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="wl_wentylacja2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Wentylacja</string>
      </property>
     </widget>
     <widget class="QLabel" name="stezenie_dymu2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dopuszczalne stężenie dymu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ustaw_temperature2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ustaw temperature:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="master_temp_wysw2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value" stdset="0">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="wl_swiatlo2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Światło</string>
      </property>
     </widget>
     <widget class="QLabel" name="aktualna_temperatura2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aktualna temperatura:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="limit_dymu2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
     <widget class="QTextBrowser" name="kontrola_dymu2">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>260</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="czujnik_ruchu2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="czujnik_drzwi2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="czujnik_okna2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>configure</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
